source/clear-linux/tutorials/multi-boot/multi-boot-cl.rst

```diff
@@ -0,0 +1,172 @@
+
+         Figure 5: |CL|: New partition size
+
+   #. Set the partition type to :guilabel:`8300 (Linux filesystem)`. See
+      Figure 6.
+
+      .. figure:: figures/multi-boot-06.png
+
+         Figure 6: |CL|: Set partition type
+
+   #. Name the partition :file:`CL-root`. This name makes it easier to
+      identify later. See Figure 7.
+
+      .. figure:: figures/multi-boot-07.png
+
+         Figure 7: |CL|: Name partition
+
+#. Create a new swap partition. See Figure 8.
+
+   .. figure:: figures/multi-boot-08.png
+
+      Figure 8: |CL|: Create swap partition
+
+   #. Select the `free space` partition located at the bottom of the column.
+
+   #. Select :guilabel:`New`. See :ref:`Figure 4<multi-boot-04>`.
+
+   #. Accept the default first sector.
+
+   #. Specify the desired size of the swap partition. For this example, we
+      used 8GB. See the `recommended swap partition sizes`_ for guidance.
+
+   #. Set the partition type to :guilabel:`8200 (Linux swap)`.
+
+   #. Name the partition :file:`CL-swap`.
+
+#. Create a new EFI partition. See Figure 9.
+
+   .. figure:: figures/multi-boot-09.png
+
+      Figure 9: |CL|: Create EFI partition
+
+   #. In the Partition Type column, select :guilabel:`free space` located at
+      the bottom of the column.
+
+   #. Select :guilabel:`New`. See :ref:`Figure 4<multi-boot-04>`.
+
+   #. Accept the default first sector.
+
+   #. Specify the desired size of the partition. For this example, we used
+      1024 MB. This partition will hold |CL|, the kernels of the other
+      operating systems, and their boot information. Its size depends on the
+      number of installed operating systems. In general, allocate about 100MB
+      per operating system. For this example, we used 1024 MB.
+
+   #. Set the partition type to :guilabel:`ef00 (EFI partition)`.
+
+   #. Name the partition :file:`CL-EFI`.
+
+#. Select :guilabel:`Write` to apply the new partition table.
+
+#. Select :guilabel:`Quit` to exit the `cgdisk` tool.
+
+#. On the :guilabel:`Set mount points` screen, specify the mount points and
+   format settings as shown in Figure 10.
+
+   .. figure:: figures/multi-boot-10.png
+
+      Figure 10: |CL|: Set mount points
+
+#. On the :guilabel:`User configuration` screen, select
+   :guilabel:`Create an administrative user`. See Figure 11.
+
+   .. figure:: figures/multi-boot-11.png
+
+      Figure 11: |CL|: User configuration
+
+#. Select :guilabel:`Add user to sudoers?`. See Figure 12.
+
+   .. figure:: figures/multi-boot-12.png
+
+      Figure 12: |CL|: Add user as sudoer
+
+#. Follow the remaining prompts to complete the installation and go through
+   the out-of-box-experience of |CL|.
+
+#. Log in.
+
+#. Get root privileges.
+
+   .. code-block:: console
+
+      $ sudo -s
+
+#. Add a timeout period for Systemd-Boot to wait, otherwise it will not
+   present the boot menu and will always boot |CL|.
+
+   .. code-block:: console
+
+      # clr-boot-manager set-timeout 20
```

Comment:
change "# clr-boot-manager set-timeout 20" to "$ sudo clr-boot-manager set-timeout 20"

Reply:
Text edits deferred to next phase of updates.